src/app/frontend/deploy/deploy.html

```diff
@@ -28,6 +28,12 @@ <h3 class="md-headline kd-deploy-form-title">Deploy a Containerized App</h3>
             Upload a YAML or JSON file
           </md-radio-button>
         </md-radio-group>
+        <kd-user-help>
+          To learn more,
+          <a href="http://kubernetes.io/v1.1/docs/user-guide/ui.html" target="_blank">
```

Comment:
Hmm.... The link here. It shows the old UI. See this announcement: https://groups.google.com/forum/#!topic/kubernetes-dev/UedwIYb-fHg

I think the correct link for 1.2 will be: http://kubernetes.github.io/docs/user-guide/ui/

What do you think?